File: java-samples/Basic/cordapp-example/contracts/src/test/java/net/corda/samples/example/contracts/StateTests.java
package net.corda.samples.example.contracts;

import net.corda.samples.example.states.IOUState;
import net.corda.testing.node.MockServices;
import org.junit.Test;

public class StateTests {
    private final MockServices ledgerServices = new MockServices();

    @Test
    public void hasAmountFieldOfCorrectType() throws NoSuchFieldException {
        // Does the message field exist?
        IOUState.class.getDeclaredField("value");
        // Is the message field of the correct type?
        assert(IOUState.class.getDeclaredField("value").getType().equals(Integer.class));
    }
}

File: kotlin-samples/Basic/opentelemetry-cordapp-example/contracts/src/test/kotlin/net/corda/samples/example/contracts/StateTests.kt
package net.corda.samples.example.contracts

import net.corda.samples.example.states.IOUState
import net.corda.testing.node.MockServices
import org.junit.Test

class StateTests {
    private val ledgerServices = MockServices()
    @Test
    @Throws(NoSuchFieldException::class)
    fun hasAmountFieldOfCorrectType() {
        // Does the message field exist?
        IOUState::class.java.getDeclaredField("value")
        assert(IOUState::class.java.getDeclaredField("value").getType() == Int::class.javaPrimitiveType)
    }
}